se.sjoerd.GIScan.yml

```diff
@@ -0,0 +1,46 @@
+app-id: se.sjoerd.GIScan
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: "5.15-21.08"
+command: runner.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=xdg-documents
+  - --device=dri
+  
+modules:  
+
+  - python3-requirements.json
+        
+  - name: GIScan
+    sources:
+      - type: git
+        url: https://github.com/SjoerdB93/GIScan.git
+        branch: main
```

Comment:
It needs to point to specific tag or commit.

Reply:
Thanks, I was unsure about this, I actually had it pointing to a specific commit at an earlier stage but going with the main branch felt more lazy/convenient.
 
Changed it to the specific commit instead.